File: beginner/C/mac_pronalts.c
#include<stdio.h>

int main(void)
{
    //controle variable
    int reptition = 0;
    int id = 0;
    int quantity = 0;
    double total = 0.0;

    scanf("%d", &reptition);

    for (int i = 0, n = reptition; i < n; i++)
    {
        //input values
        scanf("%d%d", &id, &quantity);

        switch (id)
        {
        case 1001:
            total = total + quantity * 1.50;
            break;
        case 1002:
            total = total + quantity * 2.50;
            break;
        case 1003:
            total = total + quantity * 3.50;
            break;
        case 1004:
            total = total + quantity * 4.50;
            break;
        case 1005:
            total = total + quantity * 5.50;
            break;
        default:
            break;
        }
    }

    //output
    printf("%.2f\n", total);

    return 0;
}

File: beginner/swift/macPronalts.swift
//input
var numberOfRepetition:Int = Int(readLine()!)!

var total:Double = 0

//utilizando for 
for _ in 1...numberOfRepetition{
    //inputs
    let id:Int = Int(readLine()!)!
    let quantity:Int = Int(readLine()!)!

    switch id {
        case 1001:
            total = total + (Double(quantity) * 1.50)
        case 1002:
            total = total + (Double(quantity) * 2.50)
        case 1003:
            total = total + (Double(quantity) * 3.50)
        case 1004:
            total = total + (Double(quantity) * 4.50)
        case 1005:
            total = total + (Double(quantity) * 5.50)
        default:
            print("invalid ID")
    }
}

//output
print(total)